src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -673,6 +673,47 @@ public function push()
         return true;
     }
 
+    /**
+     * Save the model and all of its relationships without throwing an Exception when parent isn't instanciated (unknown foreign key id).
+     *
+     * @param string $fk_child_name	The laravel compliant foreign key name
+     * @param mixed $fk_id			The id of the parent
+     * @return bool
+     */
+   public function pushOrCreate(?string $fk_child_name = null, $fk_id = null)
```

Comment:
camelCase needed. Also, phpdoc is wrong.

Reply:
Fixed the variable name with the commit 1938044. I tried to change the phpDoc but i am not sure what you ment by 'is wrong'. Is it about the mixed type on the $fkId?